docs/pages/database-access/guides/snowflake.mdx

```diff
@@ -36,16 +38,10 @@ This guide will help you to:
 
   See [Installation](../../installation.mdx) for details.
 
-(!docs/pages/includes/user-client-prereqs.mdx!)
-
 (!docs/pages/includes/tctl.mdx!)
 
 ## Step 1/5. Install and configure Teleport
 
-### Set up the Teleport Auth and Proxy Services
-
-(!docs/pages/includes/database-access/start-auth-proxy.mdx!)
-
 ### Set up the Teleport Database Service
```

Comment:
As noted above, could this be the H2?

Reply:
Changed. I've also changed other instances of H3s that should be H2s.